docs/spfx/library-component-tutorial.md

```diff
@@ -56,6 +56,8 @@ localization_priority: Priority
 
 ## How to consume a 3rd party SPFx library (for local testing)
 1. Run `npm link` from the root directory of library solution. In this case it would be from the **```corporate-library```** folder.
+
+1. Right after creating the symbolic link to your new spfx library component, do not forget to run at least once ```gulp serve```. You need to compile your library component at least once in order to be able to import it as a module into a different project. Remember that the package.json says that ```"main": "lib/index.js"```, therefore, that location needs to exist prior any **import** attempt.
```

Comment:
Please use proper markdown formatting. You're using fenced code formattings inline... please resolve this before we can merge.

Reply:
Do you mean to replace the code formatted strings to just use _a single quote_, for example, `gulp serve` instead of ```gulp serve``` ? or do you you mean to put any code example in a **new line?**
The rest of the documentation of this page is showing inline code examples as `npm link` or others, I just followed the format is already there.

Please give me more info, thanks

Reply:
I have done another commit with a better formatting, just a single and not triple quotes. Let me know if it is good, thanks